lib/migrations/20190530163257_track_typeOfTest.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('underscore');
+
+exports.up = async function(knex, Promise) {
+    await knex.schema.table('students', function(t){
```

Comment:
does use knex.schema.table work? or do we need to use knex.schema.alterTable just double check it

Reply:
knex.schema.alterTable it is work same as knex.schema.table is working.